v2/internal/pkg/release/signature.go

```diff
@@ -65,125 +65,134 @@ func (o SignatureSchema) GenerateReleaseSignatures(ctx context.Context, images [
 		}
 		digest := imgSpec.Digest
 
-		if digest != "" {
-			o.Log.Debug("signature digest %s", digest)
-			// check if the image is in the cache else
-			// do a lookup and download it to cache
-			sigFiles, err := os.ReadDir(o.Opts.Global.WorkingDir + SignatureDir)
-			if err != nil {
-				o.Log.Debug("[GenerateReleaseSignatures] no directory found for signatures %v", err)
-			}
-			for _, file := range sigFiles {
-				if strings.Contains(file.Name(), digest) {
-					data, err = os.ReadFile(o.Opts.Global.WorkingDir + SignatureDir + file.Name())
-					if err != nil {
-						o.Log.Warn("[GenerateReleaseSignatures] could not read %s %v", file.Name(), err)
+		// OCPBUGS-56009
+		// not worrying about code complexity as this module will eventually be deprecated
+		// in favor of the cosign signature work
+		// nolint: nestif
+		if o.Opts.Global.IgnoreReleaseSignature && len(o.Config.Mirror.Platform.Release) > 0 {
+			imgs = append(imgs, img)
+		} else {
```

Comment:
We can unindent the whole `else` block:
```suggestion
			imgs = append(imgs, img)
			continue
		}
```